packages/sdk/src/metrics/serverless.ts

```diff
@@ -89,3 +90,11 @@ export function recordServerlessResponseTime(
     "sw.transaction": transaction ?? "unknown",
   })
 }
+
+export const serverlessViews = [
+  new View({
```

Comment:
I take it the default aggregation for the serverless views is the Explicit bounds instead of Exponential?

Reply:
For the response time specifically yes, otherwise I default to exponential since they end up being a lot better representations of the data most of the time